quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/jaxrs/QuarkusKeycloakApplication.java

```diff
@@ -70,21 +80,52 @@ protected void loadConfig() {
         // no need to load config provider because we force quarkus impl
     }
 
-    private void createAdminUser() {
-        String adminUserName = getEnvOrProp(KEYCLOAK_ADMIN_ENV_VAR, KEYCLOAK_ADMIN_PROP_VAR);
-        String adminPassword = getEnvOrProp(KEYCLOAK_ADMIN_PASSWORD_ENV_VAR, KEYCLOAK_ADMIN_PASSWORD_PROP_VAR);
+    @Override
+    protected void createTemporaryAdmin(KeycloakSession session) {
+        var adminUsername = Configuration.getOptionalKcValue(BootstrapAdminOptions.USERNAME.getKey()).orElse(getEnvOrProp(KEYCLOAK_ADMIN_ENV_VAR, KEYCLOAK_ADMIN_PROP_VAR));
+        var adminPassword = Configuration.getOptionalKcValue(BootstrapAdminOptions.PASSWORD.getKey()).orElse(getEnvOrProp(KEYCLOAK_ADMIN_PASSWORD_ENV_VAR, KEYCLOAK_ADMIN_PASSWORD_PROP_VAR));
```

Comment:
Shouldn't the env vars be handled implicitly as they are Keycloak options? We won't need sys properties if admin creds can be specified as CLI args. The sys props are used just here:
https://github.com/keycloak/keycloak/blob/8eda167312fa1b54f320b4b99a442c6d3794162f/test-poc/framework/src/main/java/org/keycloak/test/framework/server/EmbeddedKeycloakTestServer.java#L16-L20

Reply:
Ok, works for me.